tests/core/test_grad_cam.py

```diff
@@ -91,6 +93,7 @@ def test_should_explain_output(mocker):
         [mocker.sentinel.image_1, mocker.sentinel.image_2],
         mocker.sentinel.layer_name,
         mocker.sentinel.class_index,
+        True
```

Comment:
Why is it `True` here and not `mocker.sentinel.use_guided_grads`?

Reply:
Sorry for my absence. I've been out for some weeks. 
I've never used `black` these mocker stuffs from pytest. I'll look for it.

Reply:
Now I think it is OK.

Should I update the `README.md` or is it the the kind of things you do by yourselves?